### PR TITLE
creating my own branch with personal page on it

### PR DESCRIPTION
**Pull request by iafhurtado**
## Description

_Concise description of proposed changes, We recommend using screenshots and videos for better description_

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #{issue number}_

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address:

### CODE REVIEW

File: package-lock.json
Comment (technophile-04) [inline package-lock.json:1]: I think you by mistake ran `npm install` instead of `yarn install`, because of which this file is created, could you please remove it 🙌

File: yarn.lock
Comment (technophile-04) [inline yarn.lock:1]: Lets revert these changes too, since we haven't updated/added any new package we can revert this file 

I think this was updated because you ran `npm install` instead of `yarn install`

Comment (iafhurtado) [reply yarn.lock:1]: Hi @technophile-04 technophile, no actually, I use windows pc to code and I always face path conflicts when I use yarn install so I'Ve been using npm install --force to get all the speedrunethereum and buidlguidl stuff working, is there another way I can do it manually to push this through?

Comment (technophile-04) [reply yarn.lock:1]: Ohh I see, But maybe I think best approach will be getting `yarn` running on your machine since a lot of projects uses it too 

could your check if this discussion https://github.com/yarnpkg/yarn/issues/1648 helps, happy to help in debugging 🙌

> is there another way I can do it manually to push this through?

You can delete `pacakage-lock.json` file and revert the changes of `yarn.lock` in new commit and push that commit

Comment (ZakGriffith) [reply yarn.lock:1]: @iafhurtado I also have a Windows PC and have been using yarn.  Can help you get yarn working in your environment too.

**Comment by iafhurtado** [comment]
I opened #20 correctly using yarn thanks to @ZakGriffith ! Will close this one if you don't mind
